src/app/entity/entity-mapper.service.ts

```diff
@@ -73,14 +73,19 @@ export class EntityMapperService {
   }
 
   public save<T extends Entity>(entity: T): Promise<any> {
-    // TODO: how to save 'references' of this Entity to other Entities?
-    //      e.g. a 'Child' may have 'FamilyMember's who are Entity instances of their own
-    //      and should be saved separately in the database
+    entity['_id'] = EntityMapperService.getDatabaseIdByEntity(entity);
```

Comment:
@sleidig I found a solution to the `_id` field problematic. We could just dynamically add this field here and thereby remove the complete database id functionality from the entity class.